### ISSUE TITLE
Feat: Add Zoom Feature to Game Screen on Scroll

### ISSUE THREAD

**Issue by gopi-trip** [EternoSeeker/gameoflife#338]
### Describe the feature

Implement a feature that allows users to zoom in and out of the game screen by scrolling. This will enhance the user experience by providing better control over the view of the game.

### Add ScreenShots

![image](https://github.com/EternoSeeker/gameoflife/assets/151914813/6d237276-4577-48c3-955a-05d80d60b9bb)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by gopi-trip** [comment]
Please assign this issue to me under gssoc'24

**Comment by EternoSeeker** [comment]
Yes that would be nice, just ensure it's well tested, thank you.
you can work on it
